IntegrationTests/tests_01_http/defines.sh

```diff
@@ -125,21 +127,25 @@ function start_server() {
 }
 
 function get_htdocs() {
+    # shellcheck source=/dev/null
     source "$1"
-    echo "$token_htdocs"
+    # shellcheck disable=SC2154
+    echo "${token_htdocs:?}"
```

Comment:
Why are we supporting it not being set? I guess the right way here would be to pass it as an argument to the function. But maybe you're trying to minimise churn?

Reply:
Yeah I didn't want to rearchitect all of this